components/tyk-dashboard/values.yaml

```diff
@@ -42,6 +42,9 @@ global:
     # The port at which the gateway service can be found
     gateway: 8080
 
+  # The name given to the service port
+  servicePortName: ""
```

Comment:
In my opinion, there are some inconsistencies regarding the location of the new fields in the values.yaml file. Some are located at the root level of components, while others are located under `<component>.service`. I would suggest moving all configurations to the `<component>.service.portName` location for consistency. What do you think?

Reply:
I notice this and I feel it has to do with the data structure being used to populate the service value. If it is a slice for instance,  it makes it easy to group the values for all the items. So, to ensure consistency, we can ensure that service items are in the slice in the helm chart as oppose to what we currently have.